### DISCUSSION TITLE
Simple error

### DISCUSSION THREAD

**Discussion by FromCppToCarbon** [carbon-language/carbon-lang#2300] [General]
I just have a COMPILATION ERROR below  ( with thanks for the answer ).

package Sanbox api;
fn Small()  {
  var x: i32 = 2;
  var y: i32 = 3;
  Print("x = {0} and y = {1}", x, y);
}
fn Main() -> i32 {
Small();
return (0);
}
------------------------
COMPILATION ERROR: ./sandbox/carbon1.carbon:9: Print takes 1 or 2 arguments, received 3
---------
Is there something missing  or   ??

**Comment by jonmeow** [comment]
The error is correct. The current incarnation of Print is only there for debug purposes, and provides only minimal support to that.

If you're instead confused about the 3 arguments, the first is the format string, the second is x, and the third is y.

#2110 is asking the leads what syntax they want for print-like functionality. Any changes are blocked on that.

**Comment by FromCppToCarbon** [comment]
Thanks a lot , it is clear now   :+1: 
it worked this way :
var x: i32 = 2;
var y: i32 = 3;
Print("x = {0}", x);
Print("y = {1}", y);
